G1ANT.Addon.Azure/Commands/KeyVaultCommand.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using G1ANT.Addon.Azure.Structures;
+using G1ANT.Language;
+
+namespace G1ANT.Addon.Azure.Commands
+{
+    [Command(Name = "azure.keyvault", Tooltip = "This command allows to access data stored in Azure Key Vault")]
+    public class KeyVaultCommand : Command
+    {
+        public class Arguments : CommandArguments
+        {
+            [Argument(Required = true, Tooltip = "Azure Key Vault client secret")]
+            public TextStructure Secret { get; set; }
+
+            [Argument(Required = true, Tooltip = "Azure Key Vault client ID")]
+            public TextStructure ClientId { get; set; }
+
+            [Argument(Required = true, Tooltip = "Azure Key Vault Url")]
+            public TextStructure Url { get; set; }
+
+            [Argument(Required = true, Tooltip = "Connection timeout to Azure in ms")]
+            public IntegerStructure AzureTimeout { get; set; } = new IntegerStructure(10000);
```

Comment:
Consider removal of the "Azure" prefix. Other properties do not use it, then why this one does?